otkafka/example_test.go

```diff
@@ -3,34 +3,41 @@ package otkafka_test
 import (
 	"context"
 	"fmt"
-	"os"
+	"strings"
 
 	"github.com/DoNewsCode/core"
+	"github.com/DoNewsCode/core/config"
 	"github.com/DoNewsCode/core/otkafka"
 	"github.com/knadh/koanf/parsers/yaml"
 	"github.com/knadh/koanf/providers/rawbytes"
 	"github.com/segmentio/kafka-go"
 )
 
 func Example_reader() {
-	var config = `
+	brokers := make([]string, len(config.EnvDefaultKafkaAddrs))
+	for i, addr := range config.EnvDefaultKafkaAddrs {
+		brokers[i] = fmt.Sprintf(`        - %s`, addr)
+	}
+	brokersStr := strings.Join(brokers, `
+`)
+	var conf = `
 log:
   level: none
 kafka:
   reader:
     default:
       brokers:
-        - ` + os.Getenv("KAFKA_ADDR") + `
+` + brokersStr + `
```

Comment:
> The purpose was to intentionally skip the test if env is mot set. （So that the test passes on any environment）

Ok, I understand. some question:
I think there should be only one place to read env. It shouldn't be read everywhere, So I added these global variables.
If possible, I can also add other non multiple envs. What do you think of this code?
```go
// Default multiple addresses from env
var (
	EnvDefaultElasticsearchAddrs = getDefaultAddrsFromEnv("ELASTICSEARCH_ADDR", "http://127.0.0.1:9200")
	EnvDefaultEtcdAddrs          = getDefaultAddrsFromEnv("ETCD_ADDR", "127.0.0.1:2379")
	EnvDefaultKafkaAddrs         = getDefaultAddrsFromEnv("KAFKA_ADDR", "127.0.0.1:9092")
	EnvDefaultRedisAddrs         = getDefaultAddrsFromEnv("REDIS_ADDR", "127.0.0.1:6379")
)
```